orttraining/orttraining/python/training/_ortmodule_graph_execution_manager.py

```diff
@@ -82,6 +82,10 @@ def __init__(self, module):
         self._save_onnx = False
         self._save_onnx_prefix = ''
 
+        # Graph transformer config
+        self._propagate_cast_ops_level = -1
+        self._propagate_cast_ops_allow = []
```

Comment:
This seems to be never populated. Is that correct?

Reply:
This line is executed in the GraphExecutionManager constructor. The user may change these values
`    model = ORTModule(model)
    model._execution_manager(True)._save_onnx = True
    model._execution_manager(True).save_onnx_prefix = "bert_fp16"
    model._execution_manager(True)._propagate_cast_ops_level = 1
    model._execution_manager(True)._propagate_cast_ops_allow = ["Gather"]
`